OPEN_PROBLEMS/DECENTRALIZED_DATA_DELIVERY_MARKETS.md

```diff
@@ -0,0 +1,509 @@
+
+At the same time, common problems addressed in the past, in traditional CDNs, reappear, albeit in a different dimension as we move to a decentralized network design space. Some of these problems are: the global orchestration of the system, the efficient allocation and use of the resources, or a clear visibility of the content being retrieved from the network.
+
+We introduce the field of Decentralized Data Delivery Markets (3DMs) to the world as a new field of research and innovation and one that has a rapidly increasing necessity of existence.  Decentralised storage networks, such as Filecoin, have reached unprecedented storage capacity commitments (of over 2.5EB of cold storage in Jan 2021) and continue to grow rapidly. There is an urgent need to complement decentralised storage with decentralised data delivery as these storage networks are seeking for a solution to deliver the data stored in their network to their end users, while meeting the expectations that end-users are used to with the centralised services of today.
+
+We envision a 3DM as being:
+
+*   A permissionless and open data delivery market, composed of one or more crypto-economic models
+*   A fair exchange relationship between service providers and users
+*   A cost efficient way for publishers to distribute their content
+*   A unique set of business opportunities including:
+    *   Delivering data to end users
+    *   Carrying data between multiple disconnected locations or with high latency between them (Data Muling and/or Package Switching)
+    *   Create ephemeral distribution networks for large gatherings
+    *   Power the next generation of Web 3.0 applications
+
+This new field is ripe with unique business opportunities given the growing demand from users for access to large datasets, videos and so on, the panoply of super-powered mobile devices that end users have access to and the limitations imposed by physics in ultimately moving increasingly large data at high speeds through fiber.
+
+In this Open Problem definition, we introduce  the multiple areas of research (or subOpenProblems), what we know so far for each one of them and link to some new ideas being explored.
+
+### Definition of the actors
+
+For convenience and shared understanding, we define here the agents within a 3DM network. These are:
+
+*   **Clients** - Fetching data from Providers
+*   **Providers** - Offer the service of data delivery to clients
+*   **Content Publishers** - The creators of content (or intermediaries) that want to have content distributed. There might be situations in which Content Publishers are willing to pay for the service consumed by Clients
```

Comment:
```suggestion
For convenience and shared understanding, we first define the agents within a 3DM network. These are:

*   **Clients** - Agents that fetch data from Providers.
*   **Providers** - Agents that offer data delivery services to Clients.
*   **Content Publishers** - Agents that create content and want to have it distributed (or intermediaries thereof). 
```

The part about publishers sometimes being willing to pay was out of place, as no one has talked about who pays at this point.